webapp/channels/src/components/admin_console/dropdown_setting.tsx

```diff
@@ -20,57 +19,49 @@ type Props = {
     helpText?: ReactNode;
 }
 
-export default class DropdownSetting extends PureComponent<Props> {
-    static propTypes = {
-        id: PropTypes.string.isRequired,
-        values: PropTypes.array.isRequired,
-        label: PropTypes.node.isRequired,
-        value: PropTypes.string.isRequired,
-        onChange: PropTypes.func.isRequired,
-        disabled: PropTypes.bool,
-        setByEnv: PropTypes.bool.isRequired,
-        helpText: PropTypes.node,
-    };
+const DropdownSetting: FC<Props> = ({
```

Comment:
The explicit type definition should not be needed. You can do something like `const Component = ({...}:Props) => {...}`.